data/hails.txt

```diff
@@ -1635,16 +1735,24 @@ phrase "friendly pirate"
 	word
 		"merchant"
 		"law enforcement"
+		"civilian trader"
+		"freelancer"
+		"bounty"
```

Comment:
"Bounty" doesn't mesh well with the phrases that follow. You wouldn't call something a "bounty convoy" or "bounty ship". "Pirate" or "outlaw" would work better.